### DISCUSSION TITLE
The Cinnamon menu does not include the standard category "Science"

### DISCUSSION THREAD

**Discussion by jose-navarro** [linuxmint/discussions#420] [Development]
Hi!

I'm developing a series of application for Linux. As a finishing touch, I like to add entries to the menu so users may start the applicacions without having to remember their precise names.

My applications fall clearly in the "Science" category. So, I prepare .desktop files like the one below: 
`
[Desktop Entry]
Name=purge_overlaps
Comment=Purge overlapping ADAs
Exec=/usr/local/bin/purge_overlaps
Icon=/usr/share/adatools/purge_overlaps.png
Terminal=false
Categories=Science;
Type=Application
`
Well, it seems that the Cinnamon menu does not support this standard category and my apps appear under the "Other" category.

Should't this be corrected? As far as I know, "Science" is one of the **standard** menu entries: `https://specifications.freedesktop.org/menu-spec/latest/apa.html`

Thanks!

**Comment by 100savage** [comment]
I believe Science falls under the Education Categories. Below is the Categories for the program Stellarium
`Categories=Astronomy;Education;Science;`

You may need something like this
```
[Desktop Entry] 
Name=purge_overlaps 
Comment=Purge overlapping ADAs 
Exec=/usr/local/bin/purge_overlaps 
Icon=/usr/share/adatools/purge_overlaps.png 
Terminal=false 
Categories=Education;Science; 
Type=Application
```

**Comment by anaximeno** [comment]
Solved in https://github.com/linuxmint/cinnamon/pull/12215
